docs/api/sql/Raster-operators.md

```diff
@@ -1548,21 +1548,28 @@ Output:
 
 ### RS_Value
 
-Introduction: Returns the value at the given point in the raster.
-If no band number is specified it defaults to 1. 
+Introduction: Returns the value at the given point in the raster. 
 
 Format: 
 
-`RS_Value (raster: Raster, point: Geometry)`
-
```

Comment:
Please keep `RS_Value (raster: Raster, point: Geometry)` signature because we don't want to introduce any API breaking change. 

Then revert the change R test because it is no longer needed.

Reply:
Have kept `RS_Value (raster: Raster, point: Geometry)` signature and reverted changes to R tests.

Note: 
- Only Point geometry variant of RS_Value and RS_Values can take band as optional parameter
- Same currently not possible for the integer variant because of conflicting function signatures as shown below,
`RS_Value (raster: Raster, colX: Integer, colY: Integer)`
`RS_Value (raster: Raster, point: Geometry, band: Integer)`

Reply:
Can you also add the explanation to the RS_Value and RS_Values doc: the input geometry point must be in the same CRS of the raster.